visitor.php

```diff
@@ -316,6 +325,20 @@ private function getElementsFromDescription(Description $tagDescription, bool $o
                 return null;
             }
 
+            $nextLevel = $level + 1;
+            $subTypes = $this->getTypesAtLevel($typeTag, $optional, $nextLevel);
+
+            if (! empty($subTypes)) {
```

Comment:
Does it mean a non-empty array?

Reply:
What is the base value for an array?

- `null`
- or `[]` ?

Do we use `null` for a special purpose?

Reply:
I'm not sure why there's both `null` and `[]` return values, I need to remind myself.

Reply:
It can be so simple: https://github.com/crwlrsoft/crawler/pull/28/files